src/configs/chalk.c42spec

```diff
@@ -453,6 +453,173 @@ failure, and no other keys will be checked.
   }
 }
 
+linguist_types := ["programming", "data", "markup"]
+object linguist_language {
+  user_def_ok:   false
+  gen_fieldname: "linguist_languages"
+  gen_typename:  "LinguistLanguage"
+  gen_setters:   false
+
+  field type {
+    type:    string
+    require: true
+    choice: linguist_types
+  }
+
+  field extension {
+    type:    string
+    require: true
+  }
+
+  field altextensions {
+    type:    list[string]
+    require: false
+  }
+
+}
+
+
+valid_tech_stack_categories := ["database", "webServer", "protocol", "framework"]
```

Comment:
most of con4m is using snake_case for things. only camelCase things in here are which get compiled to `nim` as nim is camelCase.

should we change these constants to snake case like `web_server` for consistency